config.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version='1.0' encoding='utf-8'?>
-<widget id="com.glowing_bear" version="0.5.3-b2" android-versionCode="504" xmlns="http://www.w3.org/ns/widgets" xmlns:cdv="http://cordova.apache.org/ns/1.0">
+<widget id="com.glowing_bear"  ios-CFBundleIdentifier="com.glowing-bear" version="0.5.3-b2" android-versionCode="504" xmlns="http://www.w3.org/ns/widgets" xmlns:cdv="http://cordova.apache.org/ns/1.0">
```

Comment:
it would probably be better to change this to "org.glowing-bear", given that that's our actual domain. I chose the android package name before we had a domain...